src/platform-implementation-js/platform-implementation.ts

```diff
@@ -73,16 +72,19 @@ export class PlatformImplementation extends SafeEventEmitter {
   Modal: Modal | null | undefined;
   Logger: AppLogger;
 
+  #driver: GmailDriver;
+  /**
+   * @internal only available in dev builds
+   */
+  private _members?: { driver: GmailDriver };
+
   constructor(driver: GmailDriver, appId: string, piOpts: PiOpts) {
     super();
     const { appName, appIconUrl, VERSION: LOADER_VERSION } = piOpts;
-    const members = {
-      driver,
-    };
-    memberMap.set(this, members);
+    this.#driver = driver;
 
     if (process.env.NODE_ENV !== 'production') {
-      (this as any)._members = members;
+      this._members = { driver };
     }
```

Comment:
let's axe this. (Chrome lets you see private variables in the console which addresses this use-case.)